Makefile

```diff
@@ -210,6 +211,13 @@ ffi/lodepng_h.lua: ffi-cdecl/lodepng_decl.c $(LODEPNG_DIR)
 # include all third party libs
 include Makefile.third
 
+# ===========================================================================
+# entry point for the application in OSX
+
+$(OUTPUT_DIR)/koreader: osx_loader.c
+	$(CC) -pagezero_size 10000 -image_base 100000000 \
```

Comment:
What do these flags do exactly?

Reply:
No idea. But Mike Pall says:

> If you're building a 64 bit application on OSX which links directly or indirectly against LuaJIT, you need to link your main executable with these flags:
`-pagezero_size 10000 -image_base 100000000`

in https://luajit.org/install.html

Reply:
Probably something allowing the JIT to make its nest for in-memory code generation.